kahuna/public/js/components/gr-archiver/gr-archiver.js

```diff
@@ -101,6 +112,18 @@ module.controller('grArchiverCtrl', [
                 reduce((all, items) => all.union(items)).
                 toArray();
         }
+        function undelete() {
+            ctrl.undeleting = true
```

Comment:
The linter is complaining about this 

```suggestion
            ctrl.undeleting = true;
```